bindings/python/Makefile.am

```diff
@@ -33,7 +33,7 @@ EXTRA_DIST = \
 if ENABLE_PYTHON_BINDING
 
 all-local:
-	$(PYTHON) $(srcdir)/setup.py build
+	CXXFLAGS="$(CXXFLAGS)" $(PYTHON) $(srcdir)/setup.py build
```

Comment:
and python does not find them because it leaves the make scope. I just want to know how much of the make scope should be exposed to python by adding them in front of the python call.